StripeConnect/StripeConnectTests/Internal/Webview/ScriptWebTestBase.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  ScriptWebTestBase.swift
+//  StripeConnectTests
+//
+//  Created by Chris Mays on 8/13/24.
+//
+
+@testable import StripeConnect
+import XCTest
+import WebKit
+
+class ScriptWebTestBase: XCTestCase {
+    
+    var webView: WKWebView!
+    
+    override func setUp() {
+        super.setUp()
+        webView = WKWebView(frame: .zero, configuration: .init())
+    }
+    
+    override func tearDown() {
+        webView = nil
+        super.tearDown()
+    }
```

Comment:
I'm curious, why do we need to set this to nil in tearDown? Shouldn't the garbage collector clean it up?

No need to make any changes, just interested for my own benefit.

Reply:
It was mostly for symmetry. ARC will take care of this for us.